src/DDpackage.cpp

```diff
@@ -705,37 +720,39 @@ namespace dd {
         }
     }
 
-    inline unsigned short Package::TThash(const unsigned short n, const unsigned short t, const short line[MAXN]) {
-        unsigned long i = t;
-        for (unsigned short j = 0; j < n; j++) {
-            if (line[j] == 1) {
-                i = i << (3U + j);
+    inline unsigned short Package::TThash(const std::set<Control>& controls, const unsigned short target) {
+        unsigned long i = target;
+        for (const auto& control: controls) {
+            if (control.type == Control::Type::pos) {
```

Comment:
Why does the hash function ignore negative Controls?

Reply:
Good catch. I'll fix this 👍🏼